structures/lists.py

```diff
@@ -20,25 +29,40 @@ def last_of_four_digits(numbers: []) -> []:
     original numbers, preserving the initial order.
     for example [1004, 9181, 1700, 4565] -> [4, 1, 0, 5]
     :param numbers: the original list of four digit integers
-    :type number: []
+    :type numbers: []
     :returns: a new list of last digits only
     :rtype: []
     """
     # TODO implement last_of_four_digits()
-    return []
+    last = []
+    i = 0
+    while i < len(numbers):
+        last.append(numbers[i] % 10)
+        i += 1
+    return last
 
 def round_up(numbers: []) -> []:
     """
     round_up() takes a list of DOUBLES and returns a new list of rounded INTEGERS.
     as a general rule, we only round up when the decimal is >= 5.
     for example [1.2, 3.6, 7.9, 4.1] -> [1, 4, 8, 4]
     :param numbers: the original list of doubles
-    :type number: []
+    :type numbers: []
     :returns: a new list of rounded integers
     :rtype: []
     """
     # TODO implement round_up()
-    return []
+    round = []
+    i = 0
+    while i < len(numbers):
+        if numbers[i] % 1 >= 0.5:
```

Comment:
-3 no comments, expecting a description of *why* the casting solution is acceptable